packages/core/src/utils/files.ts

```diff
@@ -1,4 +1,4 @@
-import { existsSync } from 'fs';
+import fss from 'fs';
 import fs from 'fs/promises';
```

Comment:
Could these be replaced with a single import statement from `fs-extra` since it has both async and sync methods?

Reply:
It can be, but I haven't checked if `fs-extra` works with `memfs`.